appveyor.yml

```diff
@@ -0,0 +1,49 @@
+build: false
+platform:
+  - x64
+clone_folder: c:\projects\workspace
+
+environment:
+  matrix:
+  - dependencies: lowest
+    php_ver_target: 7.0
+  - dependencies: highest
+    php_ver_target: 7.0
+  - dependencies: lowest
+    php_ver_target: 7.1
+  - dependencies: highest
+    php_ver_target: 7.1
+
+cache:
+    - '%LOCALAPPDATA%\Composer\files -> composer.lock'
```

Comment:
do we need to cache all of that? I though `'%LOCALAPPDATA%\Composer\files'` was enough